test/integration/config.spec.ts

```diff
@@ -42,15 +42,15 @@ describe('config poop', () => {
 	});
 
 	it('reads and exposes values from config.json', async () => {
-		const config = await import('~/src/config');
```

Comment:
The supervisor tests currently rely on removing the require cache but that will not work with actual `import()` statements hence I've had to change the code that relies specifically on `require` semantics to use `require`